code/modules/station_goals/bluespace_tap.dm

```diff
@@ -296,18 +285,18 @@
 			underlays += emissive_appearance(icon, "light_mask")
 
 /obj/machinery/power/bluespace_tap/proc/get_icon_state_number()
-	switch(input_level)
+	switch(mining_power)
 		if(0)
 			return 0
-		if(1 to 3)
+		if(50 KW to 3 MW)
 			return 1
-		if(4 to 8)
+		if(4 MW to 8 MW)
 			return 2
-		if(9 to 11)
+		if(9 MW to 11 MW)
```

Comment:
I'm pretty sure that if you have 8.1 MW, you'll fall out of this. Make sure the full range is captured.

Reply:
When above 1MW mining power is rounded down to the nearest MW

Reply:
I suppose it wouldn't matter if I did capture the full range anyhow